### DISCUSSION TITLE
Cache solution

### DISCUSSION THREAD

**Discussion by cvillami** [jhipster/jhipster-dotnetcore#738] [General]
Hi,

Can someone recommend a cache solution that I can integrate and use freely? Or if within the jhipster-dotnetcore project you are thinking of incorporating a ...

Thanks.

**Comment by nicolas63** [comment]
Hi,

You can use response cache middleware (https://docs.microsoft.com/en-us/aspnet/core/performance/caching/middleware?view=aspnetcore-5.0)

Or if you want distributed caching you can add redis cache like this (https://medium.com/net-core/in-memory-distributed-redis-caching-in-asp-net-core-62fb33925818)

**Comment by ivanmonteiro** [comment]
There is also the possibility of using NGINX as a reverse proxy (https://docs.microsoft.com/pt-br/aspnet/core/host-and-deploy/linux-nginx?view=aspnetcore-5.0) and configure NGINX's caching:
https://docs.nginx.com/nginx/admin-guide/content-cache/content-caching/
